server/app.js

```diff
@@ -46,16 +46,16 @@ app.get("/movies/:id", (req, res) => {
 
 
 tvShow = [
-    { id: 1, title: "Game Of Thrones", link: "https://en.wikipedia.org/wiki/Game_of_Thrones" },
```

Comment:
could've included the descriptive text present underneath each link

Reply:
Working on it right now :)